app/models/group.rb

```diff
@@ -692,7 +692,6 @@ def remove(user)
     has_webhooks = WebHook.active_web_hooks(:group_user)
     payload = WebHook.generate_payload(:group_user, group_user, WebHookGroupUserSerializer) if has_webhooks
     group_user.destroy
-    user.update_attribute(:primary_group_id, nil) if user.primary_group_id == self.id
```

Comment:
Will you be able to provide context on why this line is no longer required as it is not clear at the moment.